Makefile

```diff
@@ -30,8 +30,13 @@ format:
 	black -l 100 $(python-packages)
 	isort $(python-packages)
 
-test:
-	python3 -m pytest -n$(shell nproc) -vv tests
+test: test_unit test_integration
+
+test_unit:
+	python3 -m pytest -vv tests
+
+test_integration: install_devel
```

Comment:
Do we really need to re-install the development tools before each integration test? IMHO `install_devel` shouldn't be a requirement of any target. I would not be happy about the development tools being installed if I forgot to activate the venv.

Reply:
An alternative is to install `pylint` and our plugin into a temporary `venv`. I changed that accordingly - please have a look.